apps/designer/app/designer/features/sidebar-left/panels/asset-manager/asset-info-tigger.tsx

```diff
@@ -41,16 +40,19 @@ export const AssetTooltip = ({
         title="Options"
         onClick={() => setTooltipOpen(true)}
         css={{
-          opacity: 0,
+          visibility: "var(--info-trigger-visibility)",
           position: "absolute",
+          color: "$slate11",
           top: "$1",
           right: "$1",
           cursor: "pointer",
-          color: "$hiContrast",
           transition: "opacity 100ms ease",
+          "&:hover": {
+            color: "$hiContrast",
+          },
         }}
       >
-        <GearIcon />
+        <GearIcon fill={String(theme.colors.loContrast)} />
```

Comment:
I am taking the css variable from the theme, passing it as a prop to the component. Alternative would be passing it from the parent  as a new css variable like --gear-icon-fill .... not sure what is cleaner ... thoughts?